openstreetmap-carto-hstore-only-l10n.lua

```diff
@@ -521,7 +521,7 @@ function osm2pgsql.process_node(object)
     end
         
     if ((object.tags['name'] ~= nil) or (object.tags['name:' .. lang] ~= nil)) then
-        names = osml10n.get_names_from_tags(object.id, object.tags, true, false, lang, object.get_bbox)
```

Comment:
If `get_names_from_tags()` gets `object` as parameter instead of `object.get_bbox`, wouldn't it make sense to also remove the  `object.id` and `object.tags` parameters and just use the `object` inside the function?